APMrover2/Rover.h

```diff
@@ -367,6 +367,9 @@ class Rover : public AP_HAL::HAL::Callbacks {
     // set if the users asks for auto reverse
     bool in_auto_reverse;
 
+    // true if pivoting (set by use_pivot_steering)
+    bool pivot_steering_active;
```

Comment:
Thanks for reviewing this.
Yes, it should be used as the return in the use_pivot_steering() function.